src/components/VideoAudioStories.js

```diff
@@ -101,26 +98,9 @@ function VideoAudioStories(props) {
             Video & Audio Stories
           </SectionTitle>
         </Grid>
-        <Hidden smDown>
-          <Grid item>
-            <div className={classes.channel}>
-              <Typography
-                variant="subtitle2"
-                className={classes.channelCallToAction}
-              >
-                Visit our <span className={classes.channelName}>Youtube</span>{' '}
-                channel.
-              </Typography>
-              <FontAwesomeIcon icon={faYoutube} color="#023256" size="2x" />
-              <div />
-            </div>
-          </Grid>
-        </Hidden>
```

Comment:
Since we're removing channel component altogether, we can remove the whole `Grid` with it. It was there to layout the two components but now that we only have one component, we no longer need the `Grid` around it.

```jsx
...
      <Grid container justify="space-between" className={classes.title}>
        <Grid item>
          <SectionTitle subtitle="Video & Audio Stories">
            Video & Audio Stories
          </SectionTitle>
        </Grid>
      </Grid>
...
```

Can just become
```jsx
      <SectionTitle subtitle="Video & Audio Stories">
        Video & Audio Stories
      </SectionTitle>